build.sbt

```diff
@@ -84,8 +84,8 @@ releaseProcess := Seq[ReleaseStep](
 )
 
 // bintray config
-bintrayOrganization := Some("sbt")
-bintrayRepository := "sbt-plugin-releases"
+//bintrayOrganization := Some("sbt")
+//bintrayRepository := "sbt-plugin-releases"
```

Comment:
This is not mergable. Native-packager is published to the sbt-plugin-releases bintray repo.